xbmc/addons/addoninfo/AddonInfo.cpp

```diff
@@ -129,7 +129,8 @@ TYPE CAddonInfo::TranslateSubContent(const std::string& content)
 
 CAddonInfo::CAddonInfo(std::string id, TYPE type)
   : m_id(std::move(id)),
-    m_mainType(type)
+    m_mainType(type),
+    m_isBinary(false)
```

Comment:
Don't we prefer Init in the header for new code?

Reply:
changed